dozer/cogs/moderation.py

```diff
@@ -287,6 +296,8 @@ async def on_message(self, message):
     @Cog.listener('on_message_delete')
     async def on_message_delete(self, message):
         """When a message is deleted, log it."""
+        if message.author == self.bot.user:
```

Comment:
Open question: do we only want to not record deletes from Dozer, or generalize to all bots?

Reply:
Recording other bot deletions is probably a good thing